esmf_regrid/_esmf_sdo.py

```diff
@@ -185,17 +198,19 @@ def __init__(
         )
 
     def _as_esmf_info(self):
-        shape = np.array(self._shape)
+        shape = np.array(self._extended_shape)
 
         londims = len(self.lons.shape)
 
         if londims == 1:
             if self.circular:
-                adjustedlonbounds = self.lonbounds[:-1]
+                adjustedlonbounds = self._refined_lonbounds[:-1]
             else:
-                adjustedlonbounds = self.lonbounds
+                adjustedlonbounds = self._refined_lonbounds
             centerlons, centerlats = np.meshgrid(self.lons, self.lats)
-            cornerlons, cornerlats = np.meshgrid(adjustedlonbounds, self.latbounds)
+            cornerlons, cornerlats = np.meshgrid(
```

Comment:
I'm not sure it makes sense to refine a curvilinear grid since they don't really have "lines of constant latitude" in the same way.

Reply:
OK. This is also clearer to me now since you removed `2D` from the new docstring.

Reply:
with that said, I suppose it still makes sense to replicate this since we probably still want `refined_lonbounds`/`refined_latbounds` to represent the "source of truth" for ESMF representations.